### DISCUSSION TITLE
How to download minSdkVersion android version with capacitor?

### DISCUSSION THREAD

**Discussion by Nourux** [ionic-team/capacitor#3642] [General] [answered]
Hi, I am needing to download the minSdkVersion version so that more of my users can access the app, this is due to the public it is aimed at, but I still can't find a way to do this

**Comment by distante** [comment, marked as answer]
Capacitor just supports Android 5+ since it works with newish WebView/Chrome features. For it to work with old Android version you should have something like the old Cordova Cakewalk which would give your app at least 50MB of size.

So, you can not do this.

**Reply by Nourux** [reply]
Ok, great. I really appreciate it.
